linkerd/app/outbound/src/http/logical/policy/tests.rs

```diff
@@ -114,6 +147,20 @@ async fn header_based_route() {
 
     // Hold the router to prevent inner services from being dropped.
     drop(router);
+
+    let report = linkerd_app_core::metrics::FmtMetrics::as_display(&metrics).to_string();
+    let mut lines = report
+        .lines()
+        .filter(|l| !l.starts_with('#'))
+        .collect::<Vec<_>>();
+    lines.sort();
+    assert_eq!(
+        lines,
+        vec![
+            r#"outbound_http_route_backend_requests_total{parent_group="core",parent_kind="Service",parent_namespace="ns",parent_name="papa",parent_port="7979",parent_section_name="",route_group="policy.linkerd.io",route_kind="HTTPRoute",route_namespace="ns",route_name="default",backend_group="core",backend_kind="Service",backend_namespace="ns",backend_name="default",backend_port="8080",backend_section_name=""} 1"#,
```

Comment:
nit, not actually important: it seems like it might be nicer to use multi-line string literals here with `\` to escape whitespace? for example, this is the same string:
```suggestion
         "outbound_http_route_backend_requests_total{\
              parent_group=\"core\",\
              parent_kind=\"Service\",\
              parent_namespace=\"ns\",\
              parent_name=\"papa\",\
              parent_port=\"7979\",\
              parent_section_name=\"\",\
              route_group=\"policy.linkerd.io\",\
              route_kind=\"HTTPRoute\",
              route_namespace=\"ns\",\
              route_name=\"default\",\
              backend_group=\"core\",\
              backend_kind=\"Service\",\
              backend_namespace=\"ns\",\
              backend_name=\"default\",\
              backend_port=\"8080\",\
              backend_section_name=\"\"\
          } 1",
```
but the label values are a little easier to read...

Reply:
yeah this test could definitely be prettier in a few ways. going to leave the mush of labels for now.